app/webpack/taxa/show/components/interactions_tab.jsx

```diff
@@ -84,16 +84,7 @@ const InteractionsTab = ( { interactions } ) => {
         <Col xs={4}>
           <h3>About Interactions</h3>
           <p>
-            Most organisms interact with other organisms in some way or
-            another, and how they do so usually defines how they fit into an
-            ecosystem. These intereactions come to us
-            from
-            <a href="http://www.globalbioticinteractions.org/">Global Biotic Interactions (GLoBI)</a>,
-            a database and webservice that combines
-            interaction data from numerous sources, including iNaturalist.
-            You can actually contribute to this database by adding the
-            "Eating", "Eaten by", and "Host" observation fields to
-            observations that demonstrate those interactions.
+            Most organisms interact with other organisms in some way or another, and how they do so usually defines how they fit into an ecosystem. These interactions come to us from <a href="http://www.globalbioticinteractions.org/">Global Biotic Interactions (GLoBI)</a>, a database and webservice that combines interaction data from numerous sources, including iNaturalist. You can actually contribute to this database by adding the "Eating", "Eaten by", and "Host" observation fields to observations that demonstrate those interactions.
```

Comment:
Hopefully this set off an eslint alarm for you about the line being too long. FWIW, I think you want to change

```
ecosystem. These intereactions come to us
from
<a href="http://www.globalbioticinteractions.org/">Global Biotic Interactions (GLoBI)</a>,
```
to
```
ecosystem. These interactions come to us
from <a href="http://www.globalbioticinteractions.org/">Global Biotic Interactions (GLoBI)</a>,
```

React is weird about spacing.

Reply:
that is done, thanks